packages/api/src/ITooltip.css

```diff
@@ -15,65 +15,62 @@
 
 /* Creates a small triangle extender for the tooltip */
 .d3-tip:after {
+  content: " ";
```

Comment:
Is this needed (its not in the other ones?)

Reply:
Yes, the browser won't display pseudo elements with empty content.